<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPrincipal</class>
 <widget class="QWidget" name="MenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #2c3e50, stop:1 #3498db);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QFrame" name="frameBotones">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(255,255,255,0.1);
border: 2px solid rgba(255,255,255,0.2);
border-radius: 15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnJugar">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #27ae60, stop:1 #2ecc71);
    border: 2px solid #219653;
    border-radius: 15px;
    color: white;
    font-family: 'Arial';
    font-size: 18px;
    font-weight: bold;
    padding: 15px 30px;
    margin: 8px;
    min-width: 200px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #229954, stop:1 #27ae60);
    border: 2px solid #1e8449;
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #1e8449, stop:1 #229954);
}</string>
        </property>
        <property name="text">
         <string>JUGAR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCargarPartida">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #27ae60, stop:1 #2ecc71);
    border: 2px solid #219653;
    border-radius: 15px;
    color: white;
    font-family: 'Arial';
    font-size: 18px;
    font-weight: bold;
    padding: 15px 30px;
    margin: 8px;
    min-width: 200px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #229954, stop:1 #27ae60);
    border: 2px solid #1e8449;
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #1e8449, stop:1 #229954);
}</string>
        </property>
        <property name="text">
         <string>CARGAR PARTIDA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCreditos">
        <property name="styleSheet">
         <string notr="true">#btnCreditos {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #9b59b6, stop:1 #8e44ad);
    border: 2px solid #8e44ad;
    border-radius: 15px;
    color: white;
    font-family: 'Arial';
    font-size: 18px;
    font-weight: bold;
    padding: 15px 30px;
    margin: 8px;
    min-width: 200px;
}
#btnCreditos:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #8e44ad, stop:1 #9b59b6);
}</string>
        </property>
        <property name="text">
         <string>CREDITOS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSalir">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #27ae60, stop:1 #2ecc71);
    border: 2px solid #219653;
    border-radius: 15px;
    color: white;
    font-family: 'Arial';
    font-size: 18px;
    font-weight: bold;
    padding: 15px 30px;
    margin: 8px;
    min-width: 200px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #229954, stop:1 #27ae60);
    border: 2px solid #1e8449;
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #1e8449, stop:1 #229954);
}

#btnSalir {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #c0392b, stop:1 #e74c3c);
    border: 2px solid #a93226;
}

#btnSalir:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #a93226, stop:1 #c0392b);
}</string>
        </property>
        <property name="text">
         <string>SALIR</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frameTitulo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #e74c3c, stop:0.5 #e67e22, stop:1 #f1c40f);
border: 3px solid #c0392b;
border-radius: 20px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblTitulo">
        <property name="styleSheet">
         <string notr="true">color: white;
font-family: 'Arial Rounded MT Bold', 'Arial', sans-serif;
font-size: 48px;
font-weight: bold;
text-shadow: 3px 3px 5px rgba(0,0,0,0.5);</string>
        </property>
        <property name="text">
         <string>JUEGO DE LA OCA</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
